### PR TITLE
Support register validator for external signer

### LINKED ISSUE #5644: [MEV Boost\Builder] ValidatorRegistration signature - web3signer integration
We should add Web3Signer support as soon as they implement it on their side.

### PR DESCRIPTION
**Pull request by wcgcyx**
Signed-off-by: wcgcyx <wcgcyx@gmail.com>

## PR Description
This PR adds support for signing register validator using external signer 

## Fixed Issue(s)
fixes #5644

### CODE REVIEW

File: validator/client/src/main/java/tech/pegasys/teku/validator/client/signer/SignType.java [hunk 1]
```diff
@@ -35,5 +35,8 @@ public enum SignType {
   @JsonProperty("sync_committee_selection_proof")
   SYNC_COMMITTEE_SELECTION_PROOF,
   @JsonProperty("sync_committee_contribution_and_proof")
-  SYNC_COMMITTEE_CONTRIBUTION_AND_PROOF
+  SYNC_COMMITTEE_CONTRIBUTION_AND_PROOF,
+
```
Comment (usmansaleem) [inline validator/client/src/main/java/tech/pegasys/teku/validator/client/signer/SignType.java]: nit: extra LF

**Review by usmansaleem — APPROVED** [review]
LGTM

File: validator/client/src/main/java/tech/pegasys/teku/validator/client/signer/SignType.java (hunk 1 above)
Comment (wcgcyx) [reply validator/client/src/main/java/tech/pegasys/teku/validator/client/signer/SignType.java]: fixed

**Review by tbenr — APPROVED** [review]
LGTM.

**Pull request merged**